src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private int index = new Random().nextInt(Color.values().length);
+
     public String getRandomColor() {
-        return null;
+        Color [] arrayColors = Color.values();
+        return arrayColors[index].name();
```

Comment:
this implementation will give us the same color each time, let's have a `Random` itself as a field and call `nextInt()` in `getRandomColor()` method instead

Reply:
not fixed

Reply:
this implementation will NOT gives us the same color each time.
![image](https://user-images.githubusercontent.com/108542576/212989617-5681da7a-7333-4a78-8c9d-7a98420cc3af.png)